siliconcompiler/tools/openroad/openroad.py

```diff
@@ -34,7 +34,8 @@ def setup_tool(chip, exit=True, clobber=True):
 
     chip.set('tool', tool, 'exe', tool)
     chip.set('tool', tool, 'vswitch', '-version')
-    chip.set('tool', tool, 'version', '>=v2.0-10708', clobber=clobber)
+    # Temporarily disable OpenROAD version check
+    # chip.set('tool', tool, 'version', '>=v2.0-10708', clobber=clobber)
```

Comment:
This was necessary to get sc working with OpenROAD from conda, because those binaries only have the git hash as the version string (although the commit is from version 2.x). I'll see if I can find the reason for that over at OpenROAD.

Reply:
I think this is an issue with the openroad compile for conda (if I recall, they are using an old version of git which doesn't support the git describe call used). I think the solution would be to correct it with conda (whomever is maintaining the openroad compile).

Reply:
I'm trying to get the OpenROAD build fixed over at hdl/conda-eda, then this should hopefully be taken care of (along with the fmt issue below)